pallets/api/src/fungibles.rs

```diff
@@ -0,0 +1,84 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub use pallet::*;
+
+#[cfg(test)]
+mod mock;
+
+#[cfg(test)]
+mod tests;
+
+#[cfg(feature = "runtime-benchmarks")]
+mod benchmarking;
+
+#[frame_support::pallet]
+pub mod pallet {
+	use frame_support::{
+		dispatch::WithPostDispatchInfo, pallet_prelude::*, traits::fungibles::Inspect,
+	};
+	use frame_system::pallet_prelude::*;
+	use pallet_assets::WeightInfo;
+	use sp_runtime::traits::StaticLookup;
+
+	type AccountIdLookupOf<T> = <<T as frame_system::Config>::Lookup as StaticLookup>::Source;
+	type AssetIdOf<T> = <pallet_assets::Pallet<T, TrustBackedAssetsInstance> as Inspect<
+		<T as frame_system::Config>::AccountId,
+	>>::AssetId;
+	type AssetIdParameterOf<T> =
+		<T as pallet_assets::Config<TrustBackedAssetsInstance>>::AssetIdParameter;
+	type Assets<T> = pallet_assets::Pallet<T, TrustBackedAssetsInstance>;
+	type AssetsWeightInfo<T> = <T as pallet_assets::Config<TrustBackedAssetsInstance>>::WeightInfo;
+	type BalanceOf<T> = <pallet_assets::Pallet<T, TrustBackedAssetsInstance> as Inspect<
+		<T as frame_system::Config>::AccountId,
+	>>::Balance;
+	// Should be defined in primitives.
```

Comment:
Should be configurable via the pallet config trait rather

Reply:
Great suggestion!